task1_list_practice.py

```diff
@@ -0,0 +1,24 @@
+"""
+Task2.
+
+Usages of list comprehensions.
+"""
+
+list1 = ['ab', 'ac', 'ad', 'bb', 'bc', 'bd']
```

Comment:
it's not list comprehension :) list comprehension is smth like this `[ i*j for i in range(2) for j in range(3)] `(it's a little tip :))